src/main/java/co/elastic/logstash/filters/elasticintegration/EventProcessorBuilder.java

```diff
@@ -122,6 +122,7 @@ public EventProcessorBuilder() {
                 org.elasticsearch.ingest.common.SetProcessor.TYPE,
                 org.elasticsearch.ingest.common.SortProcessor.TYPE,
                 org.elasticsearch.ingest.common.SplitProcessor.TYPE,
+                "terminate", // note: upstream constant is package-private
```

Comment:
`TerminateProcessor` is also package private like `dot_expander` https://github.com/elastic/logstash-filter-elastic_integration/blob/main/src/main/java/co/elastic/logstash/filters/elasticintegration/EventProcessorBuilder.java#L103